include/ck_tile/ops/gemm/block/block_gemm_areg_bsmem_creg_v1.hpp

```diff
@@ -35,13 +35,16 @@ struct BlockGemmARegBSmemCRegV1
                 std::is_same_v<CDataType, remove_cv_t<typename CBlockTensor::DataType>>,
             "wrong!");
 
-        constexpr index_t MPerBlock = ABlockTensorTmp{}.get_lengths()[number<0>{}];
-        constexpr index_t NPerBlock = BBlockWindowTmp{}.get_window_lengths()[number<0>{}];
-        constexpr index_t KPerBlock = ABlockTensorTmp{}.get_lengths()[number<1>{}];
+        // constexpr index_t MPerBlock = ABlockTensorTmp{}.get_lengths()[number<0>{}];
```

Comment:
tensor shape no longer match the `kM`/`kN`/`kK`?

Reply:
https://github.com/ROCm/composable_kernel-internal/pull/32#discussion_r1480970179